api/handlers/blockinfo_handler.go

```diff
@@ -253,6 +253,24 @@ func getBeginAndEndByPage(total, p, step uint64) (page, begin, end uint64) {
 	return page, begin, end
 }
 
+func getAccountBeginAndEndByPage(total, p, step uint64) (page, begin, end uint64) {
+	totalPages := uint64(math.Ceil(float64(total) / float64(step)))
+	page = p
+	if page > (totalPages - 1) {
```

Comment:
if totalPages  == 0, the page will be -1.

Reply:
Page type is uint64